fearless-utils/src/main/java/jp/co/soramitsu/fearless_utils/wsrpc/SocketService.kt

```diff
@@ -31,13 +33,15 @@ class SocketService(
     private val logger: Logger,
     private val webSocketFactory: WebSocketFactory,
     private val reconnector: Reconnector,
-    private val requestExecutor: RequestExecutor
+    private val requestExecutor: RequestExecutor,
```

Comment:
looks like a typo, not sure if it will compile with the trailing comma

Reply:
Thanks for noting. It will compile - traling commas are supported in kotlin. It can be seen by passed CI